tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -5085,16 +5085,16 @@ Schedule:Compact,
     state->dataUnitarySystems->HeatingLoad = true;
     state->dataUnitarySystems->CoolingLoad = false;
     thisSys.setOnOffMassFlowRate(*state, OnOffAirFlowRatio, PartLoadRatio);
-    EXPECT_EQ(0.6, state->dataUnitarySystems->CompOffMassFlow);
+    EXPECT_EQ(1.2, state->dataUnitarySystems->CompOffMassFlow);
     EXPECT_EQ(1.2, state->dataUnitarySystems->CompOnMassFlow);
-    EXPECT_EQ(0.6, state->dataHVACGlobal->MSHPMassFlowRateLow);
+    EXPECT_EQ(1.2, state->dataHVACGlobal->MSHPMassFlowRateLow);
     EXPECT_EQ(1.2, state->dataHVACGlobal->MSHPMassFlowRateHigh);
```

Comment:
This unit test should have noticed the issue with on and off air flow. This is correct now.